components/kyma-environment-broker/internal/health/health.go

```diff
@@ -0,0 +1,28 @@
+package health
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type HealthHandler struct {
+	Address string
+}
+
+func NewHandler(addr string) *HealthHandler {
+	return &HealthHandler{Address: addr}
+}
+
+func (h *HealthHandler) Handle() {
+	healthRouter := mux.NewRouter()
+	healthRouter.HandleFunc("/healthz", livenessHandler())
+	http.ListenAndServe(h.Address, healthRouter)
+}
+
+func livenessHandler() func(w http.ResponseWriter, req *http.Request) {
```

Comment:
```suggestion
func livenessHandler() func(w http.ResponseWriter, _ *http.Request) {
```

**nit:** if you are not using given arg then it makes sense to use underscore so it will be more descriptive that his arg is ignored. Same suggestion for below function